binding-library/java/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.microsoft.azure.functions</groupId>
     <artifactId>azure-functions-java-library-rabbitmq</artifactId>
-    <version>1.0.0-beta</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Changed this to snapshot because document was only describing SNAPSHOT and other packages.